lib/resources.js

```diff
@@ -393,37 +459,15 @@ var buildElasticQuery = function (params) {
     }
   })
 
-  // Determine if the query contains fancy pants "query-string-query" stuff like:
-  //  - field indicators, e.g. subject:"American History"
-  //  - boolean operators (AND, OR)
-  //  - quoted phrases
-  // In these cases we want to pass as a "query-string-query" query ( https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html )
-  var useQueryStringQuery = params.q && /\w{3,}:[\w"\[<>]| (AND|OR) |"(\w+\s?)+"/.test(params.q)
-
-  // TODO: This seems *generally* as good as multi_match, so let's just use query_string for everything?
-  useQueryStringQuery = true
```

Comment:
The reason for this query string query detection going away is that here we were essentially ignoring the determination and building the query in the same way regardless of whether or not it looks like a QSQ.